Appraisals

```diff
@@ -0,0 +1,18 @@
+appraise "rails-4.0" do
+  gem "rails", "4.0.2"
+end
+
+appraise "rails-4.1" do
+  gem "rails", "4.1.8"
+end
+
+appraise "rails-4.2" do
+  gem "rails", "4.2.1"
+end
+
+appraise "rails-5.0" do
```

Comment:
Can we change this Appraisal to be "rails-master" and use git as the source for Rails?